onnx/version_converter/adapters/group_normalization_20_21.h

```diff
@@ -31,59 +33,59 @@ class GroupNormalization_20_21 final : public Adapter {
     shape->addInput(node->inputs()[input_id]);
     shape->insertBefore(node);
 
-    Tensor t0;
-    t0.elem_type() = TensorProto_DataType_INT64;
+    Tensor tensor_num_grp;
```

Comment:
Nit: reminder to avoid abbreviations and to spell words out, even though it can be clear that grp is group, it is unnecessarily abbreviated and adds cognitive load to readers.